fog/report/api/tests/fog_types.rs

```diff
@@ -52,17 +52,17 @@ fn prost_test_cases() -> Vec<ProstReport> {
     vec![
         ProstReport {
             fog_report_id: Default::default(),
-            report: prost_verification_report("foobar"),
+            attestation_evidence: Some(prost_verification_report("foobar").into()),
```

Comment:
For all  the tests, I kept them all as `VerificationReport`s because nothing is currently generating the `DcapEvidence`.  I'm thinking we'll want a PR after the DCAP evidence starts coming where we change all these to `DcapEvidence` variants

Reply:
Yes, that makes sense.